prortifolio.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+    <link rel="stylesheet" href="stayle.css">
+
+</head>
+<meta charset="UTF-8"> 
+<body >
+    <div class="link">
+    <a href="file:///Users/test/Sei/homeworks/hw-w02d01-website/homeworkw2d1/hw-w02d01-website/aboutMe.html">About Me | </a>
```

Comment:
use relative paths to refer to other files in your computer. this will reduce the number of errors you will have when making websites in the future.